lib/jobs/install-os.js

```diff
@@ -310,7 +286,17 @@ function installOsJobFactory(
             self._subscribeNodeNotification(self.nodeId, function(data) {
                 assert.object(data);
 
-                return self.updateGraphProgress(self.options.progressMilestones.completed)
+                return Promise.resolve()
+                .tap(function() {
```

Comment:
Since publishing progress should not affect the original flow, I think `tap` is better than `then`.
@yyscamper @pengz1 What do you think about that?